src/compliment/context.clj

```diff
@@ -18,6 +18,7 @@
          (-> s
              (str/replace "{" "(compliment-hashmap ")
              (str/replace "}" ")")
+             (str/replace #"::([-\w]+/)" (fn [[_ kw-ns]] (str ":" kw-ns)))
```

Comment:
It seems more accurate to query `(ns-aliases *ns*)` to obtain a namespace name to be prepended to `kw-ns`?